### PR TITLE
007 - Linting Checks

### PR DESCRIPTION
**Pull request by Jack-M1**
Add in linting checks to ensure our code is consistent across the UI & API applications

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,17 +21,23 @@
...
     "@types/axios": "^0.14.0",
+    "@types/eslint__js": "^8.42.3",
     "@types/express": "^4.17.21",
     "@types/express-session": "^1.18.0",
     "@types/node": "^20.16.1",
     "@types/nunjucks": "^3.2.6",
+    "dotenv": "^16.4.5",
     "eslint": "^8.57.0",
+    "eslint-config-standard": "^17.1.0",
+    "eslint-plugin-import": "^2.29.1",
```
Comment (TomBillingtonUK) [inline package.json:34]: Do you need these plugins as I don’t see them being used by the config file

Comment (Jack-M1) [reply package.json]: No I don't, they must have been remnants from the attempt at google linting. Removed them in the most recent commit.

Comment (TomBillingtonUK) [reply package.json]: There are still 2 in there, are those needed

Comment (Jack-M1) [reply package.json]: They've been removed

**Review by Leebee1392 — APPROVED** [review]

**Review by jemorak — APPROVED** [review]

**Pull request merged**